### PR TITLE
Changed default label selector

### PR DESCRIPTION
**Pull request by alpeb**
to `failover.linkerd.io/controlled-by={{.Release.Name}}`, because `app.kubernetes.io/managed-by` is taken hostage by Helm.

Also changed Helm config from `labelSelector` to `selector`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,8 +57,9 @@ CRD, it is required to install the `linkerd-smi` extension.
 
 The following Helm values are available:
 
-- `labelSelector`: determines which `TrafficSplit` instances to consider for
-  failover. It defaults to `app.kubernetes.io/managed-by=linkerd-failover`.
+- `selector`: determines which `TrafficSplit` instances to consider for
+  failover. It defaults to `app.kubernetes.io/managed-by={{.Release.Name}}`
```
Comment (hawkw) [inline README.md:61]: shouldn't this say
```suggestion
  failover. It defaults to `failover.linkerd.io/controlled-by={{.Release.Name}}`
```

**Review by hawkw — CHANGES_REQUESTED** [review]
it looks like the README now has the wrong label name in it? other than that, lgtm

File: src/main.rs [hunk 2]
```diff
@@ -29,7 +29,7 @@ struct Args {
 
     #[clap(
         long,
-        default_value = "app.kubernetes.io/managed-by=linkerd-failover",
+        default_value = "failover.linkerd.io/controlled-by=linkerd-failover",
```
Comment (olix0r) [inline src/main.rs:32]:
```suggestion
        default_value = "failover.linkerd.io/controlled-by",
```

We could just select on the label's existence by default...

**Review by olix0r — APPROVED** [review]

File: src/main.rs (hunk 2 above)
Comment (alpeb) [reply src/main.rs:32]: Ah I like that!

**Review by hawkw — APPROVED** [review]
looks good to me now, thanks!

**Pull request merged**